tendrl-node-agent.service

```diff
@@ -3,11 +3,12 @@ Description= A python agent local to every managed storage node in the sds clust
 
 [Service]
 Type=simple
+Environment="HOME=default_foo_value"
```

Comment:
The Home will here get the path /default_foo_value and the ansible_exec.... script file will be created under /default_foo_value/.tendrl/node-agent/ansible_exec..

I could not able to set the user home directory to the HOME variable. Because,
unfortunately systemd  requires full path and it could not able to expand $HOME.
for the environment variable. 

This issue can be fixed by either hard-code the value in our /etc//tendrl/node-agent/node-agent.conf.yaml file to use "/var/tmp" or we should set /root to HOME variable in service file if root is the only user going to execute.

Need your suggestion which way we can proceed.